content/changelog/2025-05-16.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Neon Data API, Neon credits, collapsible console sidebar, and more
+---
+
+## Introducing the Neon Data API (Early Access)
+
+We're excited to introduce the **Neon Data API**, now available in **Private Preview**.
+
+<Admonition type="comingSoon" title="Request Access">
+What to try it out? Message us from the [Console](https://console.neon.tech/app/projects?modal=feedback) or on [Discord](https://discord.com/channels/1176467419317940276/1176788564890112042) and we'll send you an invite.
+</Admonition>
+
+This feature provides a RESTful API on top of your Neon Postgres database, powered by [PostgREST](https://postgrest.org). It enables direct access to tables, views, and functions using standard HTTP verbs—ideal for client-side applications and rapid prototyping.
+
+With the Neon Data API, you can:
+
+- Query tables with HTTP requests
+- Use PostgREST-compatible SDKs like `postgrest-js`
+- Scale the API alongside your Neon database
+- Secure access with Neon Auth or bring your own JWKS (e.g., Auth0, Clerk)
+
+**Enabling the Data API**
+
+You can enable the API from the Neon Console or programmatically via our new `POST /data-api` endpoint. Each branch can have its own API instance.
+
+Example request:
+
+```bash
+curl --location 'https://console.neon.tech/api/v2/projects/<project_id>/branches/<branch_id>/data-api'
+  --header 'Authorization: Bearer <token>'
+  --data '{ "database_name": "mydb" }'
+```
+
+**Authentication**
+
+By default, Neon Auth is provisioned with your API. You can also configure your own `jwks_url` for third-party auth providers:
+
+```bash
+curl --location 'https://console.neon.tech/api/v2/projects/<project_id>/branches/<branch_id>/data-api'
+  --header 'Authorization: Bearer <token>'
+  --data '{ "jwks_url": "https://your-auth.com/.well-known/jwks.json" }'
+```
+
+**Getting started**
+
+Once enabled, your REST endpoint is ready in about 15 seconds. Here's a sample request:
+
+```bash
+curl --location 'https://<your-api-url>/playing_with_neon'
+     --header 'Accept: application/json'
+     --header 'Authorization: Bearer <jwt>'
+```
```

Comment:
As I mentioned in the guide, I don't think we should mention the API at all, just the UI.

Reply:
ok! cc @bgrenon